simtools/camera_efficiency.py

```diff
@@ -262,41 +266,54 @@ def analyze(self, export=True, force=False):
         self._has_results = True
 
         print("\33[40;37;1m")
-        self._logger.info(f"Spectrum weighted reflectivity: {self.calc_reflectivity():.4f}")
-        self._logger.info(
-            f"Camera nominal efficiency with gaps (B-TEL-1170): {self.calc_camera_efficiency():.4f}"
-        )
-        self._logger.info(
-            "Telescope total efficiency"
-            f" with gaps (was A-PERF-2020): {self.calc_tel_efficiency():.4f}"
-        )
-        self._logger.info(
-            (
-                "Telescope total Cherenkov light efficiency / sqrt(total NSB efficency) "
-                "(A-PERF-2025/B-TEL-0090): "
-                f"{self.calc_tot_efficiency(self.calc_tel_efficiency()):.4f}"
-            )
-        )
-        self._logger.info(
-            "Expected NSB pixel rate for the reference NSB: "
-            f"{self.calc_nsb_rate():.4f} [p.e./ns]"
-        )
+        self._logger.info(f"\n{self.results_summary()}")
         print("\033[0m")
 
         if export:
             self.export_results()
 
     # END of analyze
 
+    def results_summary(self):
+        """
+        Print a summary of the results.
+        Include a header for the zenith/azimuth settings and the NSB spectrum file which was used.
+        The summary includes the various CTAO requirements and the final expected NSB pixel rate.
+        """
+        summary = (
+            f"Results summary for {self._telescope_model.name} at "
+            f"zenith={self.config.zenith_angle:.1f}, "
```

Comment:
zenith and azimuth are not appearing with units in the summary message. As the reflectivity and efficiency have no units, I would suggest adding the units to the angles

Reply:
Added units (manually, but they are converted to degrees so it's fine)